docs/beta/changes.md

```diff
@@ -106,6 +106,40 @@ to:
 gauge(self, name, value, tags=None, hostname=None, device_name=None)
 ```
 
+### Docker check
+
+The Docker checks is being rewritten in Go to take advantage of the new
+internal architecture of the Agent, mainly bringing a consistent behaviour
+across every container related component. Therefore the Python versions will
+never work within Agent 6. The rewrite is not yet finished, but the new
+`docker` check offers basic functionalities.
+
+The new check is named `docker` and no longer `docker_daemon`.
+
+For now we support a subset of metrics, docker events and `docker.status`
+service check. Look into
+[`docker.yaml.example`](/pkg/collector/dist/conf.d/docker.yaml.example) for
+more information.
+
+Main changes:
+
+- Some option has been moved from `docker_daemon.yaml` to the main `datadog.yaml`.
```

Comment:
rather `some options have`? If there's just one we can just name it here.

Reply:
The next block is the list of options that have moved